_sass/support/_variables.scss

```diff
@@ -151,3 +151,5 @@ $media-queries: (
   lg: $content-width + $nav-width,
   xl: 1400px,
 ) !default;
+
+$header-image: url("../../assets/images/header.png") no-repeat 50% 50%;
```

Comment:
Try to get this into html so we can use a config option & liquid

Reply:
Done.